Content/src/Client/index.html

```diff
@@ -3,12 +3,16 @@
 <head>
     <title>SAFE Template</title>
     <!--#if (Fulma != "none")
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
I think I did that recently in my code as well and html validation said it should be in a separate meta tag.